CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * [ENHANCEMENT] Add sanity check at startup to ensure the configured filesystem directories don't overlap for different components. #2828
 * [ENHANCEMENT] Go: updated to go 1.19.1. #2637
 * [ENHANCEMENT] Runtime config: don't unmarshal runtime configuration files if they haven't changed. This can save a bit of CPU and memory on every component using runtime config. #2954
+* [ENHANCEMENT] Monitoring: Add additional flags to conditionally enable log / metric scraping. #2936
```

Comment:
Please move the changelog entry to the helm chart changelog in `operations/helm/charts/mimir-distributed/CHANGELOG.md`.  See for reference: https://github.com/grafana/mimir/blob/main/docs/internal/contributing/contributing-to-helm-chart.md